.cirrus.yml

```
@@ -48,72 +48,121 @@ freebsd_environment: &FREEBSD_ENVIRONMENT
     ZEEK_CI_CPUS: 8
     ZEEK_CI_BTEST_JOBS: 8
 
-builds_only_if_template: &BUILDS_ONLY_IF_TEMPLATE
-  # Rules for skipping builds:
-  # - Do not run builds for anything that's cron triggered
-  # - Don't do darwin builds on zeek-security repo because they use up a ton of compute credits.
-  # - Always build PRs, but not if they come from dependabot
-  # - Always build master and release/* builds from the main repo
+builds_only_if_pr_master_release: &BUILDS_ONLY_IF_PR_MASTER_RELEASE
```

```suggestion
builds_only_if_pr_master_release: &ONLY_IF_PR_MASTER_RELEASE
```

This might read better where it's used. I'm not sure what `BUILDS` adds here other than probably existing already.

Also applies for the other `BUILDS_*` anchors. Not having the `BUILDS_` part would also be more consistent with the `SKIP_IF` anchoring.